opent2t.p.light/opent2t.p.light.raml

```diff
@@ -0,0 +1,295 @@
+                        "href": "/dimming",
+                        "rt": ["oic.r.light.dimming"],
+                        "if": ["oic.if.a", "oic.if.baseline"]
+                        "id": "dimming",
+                        "dimmingSetting": 30,
+                        "step": 5,
+                        "range": [0, 100]
+                      }
+                    ]
+                  }
+                ]
+              }
+
+/device/{deviceid]/power:
+  displayName: Power
+  is: [ interface ]
+
+displayName: Power
+  is: [ interface]
+
+  get:
+    responses:
+      200:
+        body:
+          application/json:
+            schema: BinarySwitch
+            example: |
+              {
+                "rt": ["oic.r.switch.binary"],
+                "id": "unique_example_id",
+                "value": false
+              }
+  post:
+    body:
+      application/json:
+        schema: BinarySwitch
+        example: |
+          {
+            "id": "unique_example_id",
+            "value": true
+          }
+    responses:
+      200:
+        body:
+          application/json:
+            schema: BinarySwitch
+            example: |
+              {
+                "id": "unique_example_id",
+                "value": true
+              }
+
+/device/{deviceid]/colourrgb:
+  displayName: Colour
+  is: [ interface ]
+
+  get:
+    responses:
+      200:
+        body:
+          application/json:
+            schema: ColourRgb
+            example: |
+              {
+                "rt": ["oic.r.colour.rgb"],
+                "id": "unique_example_id",
+                "rgbValue": [255,255,255]
+              }
+  post:
+    body:
+      application/json:
+        schema: ColourRgb
+        example: |
+          {
+            "id": "unique_example_id",
+            "rgbValue": [0,0,0]
+          }
+    responses:
+      200:
+        body:
+          application/json:
+            schema: ColourRgb
+            example: |
+              {
+                "id": "unique_example_id",
+                "rgbValue": [0,0,0]
+              }
+
+/device/{deviceid]/dimming:
+  displayName: Dimming
+  is: [ interface ]
+
+  get:
+    responses:
+      200:
+        body:
+          application/json:
+            schema: Dimming
+            example: |
+              {
+                "rt": ["oic.r.colour.rgb"],
+                "id": "unique_example_id",
+                "dimmingSetting": 30,
+                "step": 5,
+                "range": [0,100]
+              }
+  post:
+    body:
+      application/json:
+        schema: Dimming
+        example: |
+          {
+            "id": "unique_example_id",
+            "dimmingSetting": 50
+          }
+    responses:
+      200:
+        body:
+          application/json:
+            schema: Dimming
+            example: |
+              {
+                "id": "unique_example_id",
+                "dimmingSetting": 50
+              }
+
+/SubscribeLampResURI:
```

Comment:
lets change this from the old name to just /subscribe

---
In reply to: [87448818](https://github.com/openT2T/translators/pull/117#discussion_r87448818) [](ancestors = 87448818)